.gitignore

```diff
@@ -12,6 +12,8 @@
 /tmp/*
 !/log/.keep
 !/tmp/.keep
+/coverage/
```

Comment:
I was trying to get the ignore file to ignore the coverage files for git but it doesnt seem to be working, I thought that pushing the changes would work but it looks like it didn't. I suppose the changes should be reverted and not included in the pull request, sorry about that. Can make a new pr if need be.

Reply:
This is fine we will figure out git ignore soon